views/doc/technology_stack.rhtml

```diff
@@ -0,0 +1,80 @@
+<div class='technology-page'>
+  <h2><a id="technology_stack">Technology Stack</a></h2>
+
+  <div class="">
+    <span class="subtitle"><a id="programming_languages_and_their_versions">Programming languages and their versions</a></span>
+    <ul class="mt--5">
+      <li><%= "Ruby #{RUBY_VERSION}" %></li>
```

Comment:
Can we create correspondent methods for
- ruby version
- ruby on rails version
?